types/reactstrap/lib/Row.d.ts

```diff
@@ -1,8 +1,8 @@
-interface Props {
+interface Props extends React.HTMLProps<any> {
```

Comment:
I would say the right interface to extends is `React.HTMLAttributes<HTMLDivElement>`.